homeassistant/components/tplink_omada/binary_sensor.py

```diff
@@ -63,6 +60,18 @@ def get_gateway_port_status_sensors(
                     update_func=lambda p: p.wan_connected,
                 ),
             )
+            yield OmadaGatewayPortBinarySensor(
+                coordinator,
+                gateway,
+                GatewayPortBinarySensorConfig(
```

Comment:
Side note: This config should be replaced with an entity description.

https://developers.home-assistant.io/docs/core/entity#entity-description